doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+=======================================
+Harmonic Analysis via the Krylov Method
+=======================================
+
+Introduction
+============
+You can use the frequency-sweep Krylov method for a high performance solution of forced-frequency simulations 
+in acoustic or single-field structural analyses.
+
+Similar to the full harmonic analysis, the frequency sweep Krylov method also uses the full system matrices 
+to compute the harmonic response. While the full method solves the full system matrices at every frequency point 
+in the frequency range, the frequency-sweep Krylov method performs the following steps to approximate the response 
+across the frequency range :
+
+* Builds a Krylov subspace set of vectors at the frequency value in the middle of the requested frequency range,
+* Reduces the system matrices and loading on the entire frequency range,
+* Solves the reduced system, and
+* Expands the results back to compute the harmonic response. 
+
+MAPDL provides the following methods to implement a harmonic analysis using the Krylov Method:
+
+#. Via Mechanical APDL commands.
+#. Via macros, a feature described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
+#. PyMAPDL implementation of the macros, a feature described here.
+
+Assumptions
+-----------
+The following assumptions are made in the Krylov solution obtained using this method :
+
+* The stiffness, mass, and damping matrices are assumed to be constant (independent of frequency).
+
+* The external load vector is linearly ramped over frequency. Ramping assumes that the frequency at 
+  which the Krylov subspace is built is in the middle of the frequency range. If you wish to apply 
+  stepped loading, there is an option to specify that in the inputs for the 
+  :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` method.
+
+
+Krylov : PyMAPDL implementation
+===============================
+The PyMAPDL implementation gives the user customization and flexibility in terms of accessing
+the subspace vectors and reduced solutions in the python format for defining user defined routines.
+
+If you do not require customization, you can use the simpler using APDL commands to solve a harmonic analysis 
+with the Krylov method.
```

Comment:
I would say the opposite!
```suggestion
If you do require customization, you can use the simpler using APDL commands to build and solve the harmonic analysis using the Krylov method in a customized way.
```

Reply:
I don't know which of you is correct, but we need to do something about the "you can use the simpler using APDL commands" portion!

Reply:
Changed to :
"If you do not require customization, you can use the APDL commands to solve a harmonic analysis 
with the Krylov method."

Reply:
@germa89 what I Intend to convey in this statement is that user can switch to std apdl way of solve if they do not require customization.
By customization I mean to say the flexibility of the user to use pymadl to modify the subspace or vectors before running each of the steps on krylov.